extension/scripts/Selector/ConstantValue.js

```diff
@@ -0,0 +1,51 @@
+var ConstantValue = {
+
+    canReturnMultipleRecords: function () {
+        return false;
+    },
+
+    canHaveChildSelectors: function () {
+        return true;
+    },
+
+    canHaveLocalChildSelectors: function () {
+        return true;
+    },
+
+    canCreateNewJobs: function () {
+        return false;
+    },
+
+    willReturnElements: function () {
+        return false;
+    },
+
+    _getData: function (parentElement) {
+
+        var dfd = $.Deferred();
+
+        var elements = this.getDataElements(parentElement);
+
+        var result = [];
+        $(elements).each(function (k, element) {
+            $(element).val(this.constantValue);
+        }.bind(this));
```

Comment:
How can we get data elements if we do not have selector in ConstantValue as a feature?